### ISSUE TITLE
[bug]: error "module 'signal' has no attribute 'SIGKILL'" when running "python scripts/preload_models.py" on install

### ISSUE THREAD

**Issue by Slackalope1** [invoke-ai/InvokeAI#1288]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### OS

Windows

### GPU

cuda

### VRAM

8gb

### What happened?

When installing using the 'easy peasy windows install' instructions, on the "python scripts/preload_models.py" step it exits with a "module 'signal' has no attribute 'SIGKILL' "error and exits without downloading anything.

Full text of the error is attached. 

edit: I just tried the conda method to install and got the same error at the same step. If that helps.

[error.txt](https://github.com/invoke-ai/InvokeAI/files/9892570/error.txt)

### Contact Details

ken@gracidesigns.com

**Comment by FreeBlues** [comment]
You can refer to #1264 , A temporary solution was suggested:

>>I guess for now users will have to rollback and/or wait for a fix. Or, of course, patch it themselves (you can just find-and-replace signal.SIGKILL with signal.SIGTERM and it should work).

**Comment by Slackalope1** [comment]
I managed a workaround using information in the thread linked by @FreeBlues 

changing these lines in environment.yml
```

dependencies:
  - python>=3.9
  - pip>=22.2
  - cudatoolkit=11.3
  - pytorch=1.11.0
  - torchvision=0.12.0
  - numpy=1.19.2

```
and in requirements-lin-win-colab-CUDA.txt changed:

```
torch==1.11.0
--extra-index-url https://download.pytorch.org/whl/cu113
```

then rebuilt the environment with:

pew new --python 3.10 -r requirements-lin-win-colab-CUDA.txt invokeai

seemed to do the trick.

**Comment by lstein** [comment]
@tildebyte and @blessedcoolant , can you advise whether this workaround should be committed? I'm reluctant to make any substantial changes to the dependencies files.

**Comment by tildebyte** [comment]
🤷 My long-running environment has torch==1.12.1+cu116 and I have no issues. I just re-ran the preload script and likewise it's fine.
TL;DR - we probably *should* pin torch, but I don't think we should go back as far as 1.11+cu113

**Comment by lstein** [comment]
The current `environment.yml` file fixes this bug. You need to do a conda update. The requirements.txt file for the easy Peasy install needs updating. Look for a pr
